src/intervals/interval_operations/pointwise_boolean.jl

```diff
@@ -0,0 +1,206 @@
+# This file is part of the IntervalArithmetic.jl package; MIT licensed
+
+#=  This file contains functions not specified in the IEEE Std 1788-2015,
+    extending boolean operations defined for reals to intervals.
+
+    This is different from what is described in sections 9.5 of the
+    IEEE Std 1788-2015 (Boolean functions of intervals), since we deal here
+    with what is needed for drop-in replacement of floats by intervals,
+    and not what is reasonnable to do with intervals in a separate execution
+    environment.
+
+    Essentially, julia is better at composability and we are more ambitious
+    that what the standard ever expected, so we have some extra problems
+    to solve.
+
+    We use a trait system, defining the operation with an extra
+    `PointwisePolitic` argument defining how it should be handled.
+
+    The default operations use `IntervalArithmetic.pointwise_politic()`,
+    so in order to change the default behavior this function has to be
+    redefined.
+
+    We define every operations using the default `PointwisePolitic{:ternary}()`
+    and then use it to define all the others.
+=#
+"""
+    PointwisePolitic{P}
+
+Define which politic we use to extend pointwise comparison of 
+
+Valid value for the politic identifier `P` are
+    - `:is_all` : A boolean operation is extended by asking "is it true for
```

Comment:
this could be called `certainly`, the same name is used in other pre-standard interval arithmetic libraries (e.g. Gaol and Boost).